package.json

```diff
@@ -2,7 +2,7 @@
   "name": "keysim",
   "version": "1.3.0",
   "description": "Keyboard simulator for JavaScript.",
-  "main": "dist/keysim.js",
+  "main": "lib/keysim.js",
```

Comment:
Makes sense, as well. When used in node, there is no need to import the built module.